packages/sdk/tasks/deposit-bnb.ts

```diff
@@ -0,0 +1,432 @@
+            messagePassedLogs[0].args?.sender,
+            messagePassedLogs[0].args?.target,
+            messagePassedLogs[0].args?.value,
+            messagePassedLogs[0].args?.gasLimit,
+            messagePassedLogs[0].args?.data,
+          ])
+        console.log(
+          'Gas estimation for finalizeWithdrawalTransaction',
+          gasEstimation.toString()
+        )
+      }
+
+      let err = true
+      while (err) {
+        try {
+          await gasEstimationFinalSubmit()
+          err = false
+        } catch (e) {
+          console.log(
+            'Failed to get gas estimation for finalizeWithdrawalTransaction'
+          )
+          await sleep(12000)
+        }
+      }
+      const claimTokenTx = await OptimismPortal.finalizeWithdrawalTransaction([
+        messagePassedLogs[0].args?.nonce,
+        messagePassedLogs[0].args?.sender,
+        messagePassedLogs[0].args?.target,
+        messagePassedLogs[0].args?.value,
+        messagePassedLogs[0].args?.gasLimit,
+        messagePassedLogs[0].args?.data,
+      ])
+      await claimTokenTx.wait()
+      console.log(`Withdrawal finalized - ${claimTokenTx.hash}`)
+    }
+
+    console.log('Starting BOBA withdrawal')
+    const preBobaBalance = await L1BobaToken.balanceOf(l1Signer.address)
+    const withdrawalBobaTx = await L2StandardBridge.withdraw(
+      l2BobaTokenAddress,
+      utils.parseEther('1'),
+      99999,
+      '0x',
+      { value: utils.parseEther('1') }
+    )
+    const withdrawalBobaReceipt = await withdrawalBobaTx.wait()
+
+    console.log('Starting BNB withdrawal')
+    const opBalanceAfter = await signer!.provider!.getBalance(
+      OptimismPortal.address
+    )
+    const approveBnbTx = await L2BnbToken.approve(
+      L2StandardBridge.address,
+      utils.parseEther('1')
+    )
+    await approveBnbTx.wait()
+    const withdrawBnbTx = await L2StandardBridge.withdraw(
+      l2BnbTokenAddress,
+      utils.parseEther('1'),
+      99999,
+      '0x'
+    )
+    const withdrawalBnbReceipt = await withdrawBnbTx.wait()
+
+    await finalizeWithdrawal(withdrawalBobaReceipt.blockNumber)
+
+    const postBobaBalance = await L1BobaToken.balanceOf(l1Signer.address)
+    const expectedBobaBalance = preBobaBalance.add(utils.parseEther('1'))
+    if (!expectedBobaBalance.eq(postBobaBalance)) {
+      throw new Error(
+        `BOBA Balance mismatch, expected: ${expectedBobaBalance}, actual: ${postBobaBalance}`
+      )
+    }
+    console.log('BOBA Withdrawal success')
+
+    await finalizeWithdrawal(withdrawalBnbReceipt.blockNumber)
+
+    const opBalanceFinally = await signer!.provider!.getBalance(
+      OptimismPortal.address
+    )
+
+    const expectedopBalanceFinally = opBalanceFinally.add(utils.parseEther('1'))
+    if (!expectedopBalanceFinally.eq(opBalanceAfter)) {
+      throw new Error(
+        `BNB Balance mismatch, expected: ${expectedopBalanceFinally}, actual: ${opBalanceAfter}`
+      )
+    }
+    console.log('BNB Withdrawal success')
```

Comment:
I'm not seeing checks that the withdraw is reflected both on L1 and L2 as far as amounts transferred.  Probably also worth validating that the other asset (ie BNB/ETH or Boba) is not changed.